### PR TITLE
Adds Negative cache TTL

### PR DESCRIPTION
**Pull request by ankitaluthra1**
### Description
This PR adds a new hidden flag for negative cache TTL.
The value defaults to 5 secs.
The changes include unit tests and are manually verified on GKE as well.
Integration tests will be added in subsequent PR.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Yes
2. Unit tests - Yes
3. Integration tests - NA

### CODE REVIEW

File: cfg/config.go [hunk 1]
```diff
@@ -200,6 +200,8 @@ type MetadataCacheConfig struct {
 
 	TtlSecs int64 `yaml:"ttl-secs"`
 
+	NegativeTtlSecs int64 `yaml:"negative-ttl-secs"`
```
Comment (Tulsishah) [inline cfg/config.go:203]: You missed initializing flag in [params.yaml](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/cfg/params.yaml)

**Review by Tulsishah — COMMENTED** [review]
All over it looks good. Do you think we should add e2e/composite tests related to this in this PR?

File: cfg/config.go (hunk 1 above)
Comment (ankitaluthra1) [reply cfg/config.go]: added

**Review by Tulsishah — APPROVED** [review]

File: cfg/config.go [hunk 2]
```diff
@@ -196,6 +196,8 @@ type MetadataCacheConfig struct {
 
 	ExperimentalMetadataPrefetchOnMount string `yaml:"experimental-metadata-prefetch-on-mount"`
 
+	NegativeTtlSecs int64 `yaml:"negative-ttl-secs"`
```
Comment (charith87) [inline cfg/config.go:199]: Could you add a integration test for testing negative-ttl-secs.  You could maybe have a test that the 2nd read after Write expects a FileNotFound error with high negative-ttl enabled.

Comment (ankitaluthra1) [reply cfg/config.go:199]: yes I am planning to add 3 basic cases - ttl: 0, ttl -1 and default value in the subsequent PR.
